sensor_msgs/JointState joint_state
bool execute
---
trajectory_msgs/JointTrajectory traj
